pom.xml

```diff
@@ -28,6 +28,7 @@
     <pdi.version>10.3.0.0-SNAPSHOT</pdi.version>
     <pentaho-metastore.version>10.3.0.0-SNAPSHOT</pentaho-metastore.version>
     <rxjava.version>2.2.3</rxjava.version>
+    <log4j.version>2.24.1</log4j.version>
```

Comment:
@srallapa log4j version is set in maven-parent-poms. Please determine if we need to overwrite that version here

Reply:
not required, I am removing these changes and keeping only assembly changes

Reply:
updated.